src/api/TokenApi/tokenList.ts

```diff
@@ -0,0 +1,122 @@
+import { TokenDetails } from 'types'
+
+interface TokenDetailsByNetwork extends Omit<TokenDetails, 'address' | 'image'> {
+  addressByNetwork: { [networkId: number]: string }
+}
+
+// Ether, Tether, TrueUSD, USD Coin, Paxos Gemini, Dai
+const tokens: TokenDetailsByNetwork[] = [
+  // Wrapped Ether
+  //  https://weth.io
+  //  Wrapper of Ether to make it ERC-20 compliant
+  {
+    name: 'Wrapped Ether',
+    symbol: 'WETH',
+    decimals: 18,
+    addressByNetwork: {
+      1: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+      4: '0xc778417e063141139fce010982780140aa0cd5ab',
```

Comment:
Where is the network id defined? Unless this is common knowledge and they are usually referred to by id, would it make sense to create an enum for each?

Reply:
Now I saw you do have an enum for that on `src/types.ts`. Why not use that enum here as well?

Reply:
Every network has it's own id:
https://ethereum.stackexchange.com/questions/17051/how-to-select-a-network-id-or-is-there-a-list-of-network-ids

Yes, I should use the enum, I'll make the change. 
Good call!